browser_history/browsers.py

```diff
@@ -34,15 +34,17 @@ class Chrome(Browser):
 class Chromium(Browser):
     """Chromium Browser
 
-    Supported platforms (TODO: Windows and Mac OS support)
+    Supported platforms (TODO: Mac OS support)
 
     * Linux
+    * Windows
 
     Profile support: Yes
     """
     name = "Chromium"
 
     linux_path = '.config/chromium'
+    windows_path = 'AppData/Local/chromium/User Data'
```

Comment:
The PR title says "Mac OS support for Edge and Opera" but your changes are for Chromium on Windows :sweat_smile:?

Reply:
I had not touched the code for chromium browser and had added the required paths for Mac OS in Edge and Opera.
Will resolve this issue as soon as possible. I am facing some issues with merging the repo hence there might have some issues due to that.

Reply:
Right, I noticed that this happened when merging with master. You could try one of these methods:
 - copying the changes from your [first commit](https://github.com/pesos/browser-history/pull/44/commits/1369a04176652d7f1e877e243e7fbda1c9ae89a0) and committing them again
 - [interactive rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) on `upstream master` and drop the merge commits
 - delete your branch, close this PR, make your changes again and open a new PR

Reply:
Thank you very much! Will try and resolve by tonight.